internal/batches/service/service.go

```diff
@@ -380,9 +381,43 @@ func (svc *Service) ParseBatchSpec(data []byte) (*batcheslib.BatchSpec, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "parsing batch spec")
 	}
+	if err = handleMount(dir, spec); err != nil {
+		return nil, errors.Wrap(err, "parsing batch spec")
+	}
 	return spec, nil
 }
 
+func handleMount(dir string, spec *batcheslib.BatchSpec) error {
```

Comment:
This sounds like a more precise description
```suggestion
func validateMountPaths(safeDirectory string, spec *batcheslib.BatchSpec) error {
```

Reply:
Initially I went with `validate` in the function name to match the other functionality going on. But this is a bit more than "validate" It is also updating the batch spec (changing the mount path to the absolute path). So I felt `handle` was a bit closer to the functionality.

But thinking more about it, I think `validate` is "correct" since that is like 99% of the functionality